src/base/bittorrent/torrentimpl.cpp

```diff
@@ -574,6 +595,7 @@ void TorrentImpl::addTrackers(const QVector<TrackerEntry> &trackers)
 
     if (!newTrackers.isEmpty())
     {
+        m_hasWorkingTracker = hasWorkingTracker();
```

Comment:
This is pointless here. We just added new trackers so none of them are working at this point. Same goes for when replacing trackers below.